halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/providers/dockerRegistry/DockerRegistryAccountValidator.java

```diff
@@ -107,9 +110,9 @@ public void validate(ConfigProblemSetBuilder p, DockerRegistryAccount n) {
           .accountName(n.getName())
           .address(n.getAddress())
           .email(n.getEmail())
-          .password(n.getPassword())
+          .password(secretSessionManager.decrypt(n.getPassword()))
```

Comment:
you should be able to use `resolvedPassword` here, since the password is already decrypted